pep-0588.rst

```diff
@@ -10,7 +11,269 @@ Created: 27-Mar-2019
 Abstract
 ========
 
-This PEP is reserved for near future use.  Contact the authors for details.
+This PEP describes the detailed plan for migrating from Python's issue
+tracker on Roundup to GitHub issues.  See PEP 581 for rationale and
+background.  PEP 588 also describes the detailed timeline for the
+migration.
+
+
+Migration Plan
+==============
+
+Here we outline the tasks, steps, and core decisions we need to make
+in order to migrate bug tracking to GitHub, with the least impact on
+CPython developer productivity.
+
+
+Backup of GitHub data
+---------------------
+
+This effort has been started and is being tracked as an issue in
+core-workflow [#]_. We're using GitHub's Migrations API [#]_ to
+download GitHub data for CPython on a daily basis. The archives will
+be dropped in a S3 bucket.
+
+Thanks to Ernest W. Durbin III for working on this.
+
+
+Update the CLA host
+-------------------
+
+At the moment, the CLA is hosted within bpo. It needs to be updated such that
+signing the CLA does not require a bpo account, and it should be hosted outside
+of the bpo.
+
+The current CLA process itself is not ideal. Currently, contributors to
+devguide, peps, and core-workflow need to sign a CLA, and it requires a bpo
+account. A bpo account should not be required for those projects.
+
+There is an ongoing effort to start using our own instance of CLA
+assistant instead of the current CLA process in place. Discussion
+about this has been started in `core-workflow mailing list
+<https://mail.python.org/archives/list/core-workflow@python.org/thread/JBV3XJVD2DLDX5DY7TZEA6CO5YPNHJ2C/>`_
+as well as on `Discourse
+<https://discuss.python.org/t/using-cla-assistant-for-python/990>`_.
+
+
+Create Bug Triage Team on GitHub
+--------------------------------
+
+The bug triagers on bpo are valuable to the core Python workflow, and we
+definitely would need even more help with triaging issues on GitHub.
+
+It has been `proposed on Discourse
+<https://discuss.python.org/t/proposal-create-bug-triage-team-on-github/992/5>`_
+for us to create a "bug triage" team on GitHub to help with closing
+issues, notifying the appropriate parties, as well as applying labels
+to issues and pull requests. We can grant the "write" permission to
```

Comment:
This should be updated to reflect that Github now has a triage level permission https://discuss.python.org/t/pep-581-using-github-issues/535/36?u=ammaraskar

Reply:
It's a private beta though.